BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/GoogleCloudStorage/GoogleCloudStorageServiceCollectionExtensions.cs

```diff
@@ -30,6 +30,8 @@ public static void AddGoogleCloudStorage(this IServiceCollection services, Googl
             return storageClient;
         });
 
+        services.AddSingleton(_ => options);
```

Comment:
What's that?

Reply:
The Health Check needs the options to get the bucket name. In this constellation it was the only way to make the Dependency Injection work. But I can rewrite that in the `services.AddGoogleCloudStorage` method